scripts/katana_version/__main__.py

```diff
@@ -189,35 +224,118 @@ def provenance_subcommand(args):
         values.update(
             katana_enterprise_hash=git.get_hash(git.HEAD, katana_enterprise_repo_path, exclude_dirty=(SUBMODULE_PATH,))
         )
+        values.update(katana_enterprise_head=git.get_hash(git.HEAD, katana_enterprise_repo_path, pretend_clean=True))
+
+    # Sort by the key for easier reading later
+    values = dict(sorted(values.items(), key=lambda kv: kv[0]))
```

Comment:
qq: I suppose the reason for this manual json encoding is because yaml is not part of the python standard library and we wanted a uniform way to print out json, yaml, python, and `#define`-based output?

Reply:
Yes. As the commit message implies, this was not an efficient use of time. So I didn't try to optimize. I just wanted to push it to stop fiddling.